MoviesApp/MoviesApp/Services/KeychainService.swift

```diff
@@ -0,0 +1,23 @@
+// KeychainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import KeychainSwift
+
+/// Сервис хранения критических данных пользователя
+struct KeychainService: KeychainServicable {
+    // MARK: - Private properties
+
+    private static let keychain = KeychainSwift()
```

Comment:
```suggestion
    private let keychain = KeychainSwift()
```
Лучше не юзать статики а иньектить объект через конструктор